pkg/installer/deployresources.go

```diff
@@ -27,7 +27,7 @@ func (m *manager) deployResourceTemplate(ctx context.Context) error {
 
 	var installConfig *installconfig.InstallConfig
 	var machineMaster *machine.Master
-	err = pg.Get(&installConfig, &machineMaster)
+	err = pg.Get(true, &installConfig, &machineMaster)
```

Comment:
imo since it becomes critical what the value of `disallowUnknownFields` is passed in for the various cases of getting the graph we should have at least mocking expectations of the desired value. Looking through the existing tests for some of these files that would be a huge ask for this PR. WDYT about a work item being created to enhance the tests in this regard?

Reply:
Sounds like a good idea to me. More coverage is always a good thing.

Reply:
@s-amann  have you created that workitem please?